src/navigator/DrawerNavigator.tsx

```diff
@@ -305,14 +305,16 @@ export default function DrawerNavigator() {
         component={FiatExchange}
         options={{ title: t('addAndWithdraw'), drawerIcon: AddWithdraw }}
       />
-      {features.SHOW_INVITE_MENU_ITEM && (
+      {inviteMethod === InviteMethodType.ReferralUrl && (
         <Drawer.Screen
           name={'InviteModal'}
-          component={InviteFriendModal}
+          component={() =>
+            null
+          } /* Drawer.Screen requires a component to be passsed in. However, we need to use `navigate` instead. Passing in a noop. */
```

Comment:
Because the `Drawer.Screen` opens the new screen as a different screen type than was indicated in the designs. I found that the Supercharge screen in the app follows the same screen type, so I opened Invite using the same kind of logic.

I'd be happy to make a recording showing the difference in screen types, if it's helpful.

Reply:
oh yeah if you could include a recording, that'd be great! the (supercharge) ConsumerIncentivesHomeScreen appears in the drawer navigator like `component={ConsumerIncentivesHomeScreen}` rather than`component={() => null}` so i'm not sure that i follow or understand the details here :)

Reply:
I was able to get this switched to match the other screens!